test/test-kitchen.ts

```diff
@@ -71,11 +71,18 @@ test.before(async () => {
   const tarball = `${pkg.name}-${pkg.version}.tgz`;
   await renamep(tarball, `${stagingPath}/gts.tgz`);
   await ncpp('test/fixtures', `${stagingPath}/`);
-  await simpleExecp('npm install', execOpts);
 });
 
 test.serial('init', async t => {
-  await simpleExecp('./node_modules/.bin/gts init -y', execOpts);
+  const nodeVersion = Number(process.version.slice(1).split('.')[0]);
+  if (nodeVersion < 8) {
+    await simpleExecp('npm install', execOpts);
+    await simpleExecp('./node_modules/.bin/gts init -y', execOpts);
```

Comment:
Q: Any reason why there is `-y` on this leg of the branch and a `-n` on the other? If there is a valid reason, this deserves a comment.

Reply:
Added a comment.  You could use -y or -n in the < 8 version, but you must use -n in the > 8 version.